test/TestUtilities/Test.Utility/Signing/TrustedTestCert.cs

```diff
@@ -73,6 +84,57 @@ private void AddCertificateToStore()
             _store.Add(TrustedCert);
         }
 
+        //According to https://github.com/dotnet/corefx/blob/master/Documentation/architecture/cross-platform-cryptography.md#x509store,
+        //on macOS the X509Store class is a projection of system trust decisions (read-only), user trust decisions (read-only), and user key storage (read-write).
+        //So we have to run command to add certificate to System.keychain to make it trusted.
+        private void AddTrustedCert()
```

Comment:
naming: If this is only for work on Mac, the test should clearly indicate so.

Reply:
Thanks! Changed to `AddTrustedCertForMacOSX`